kim/install.sh

```diff
@@ -24,6 +24,19 @@ if [[ $? != 0 ]]; then
     exit 1
 fi
 
+# Check for tools used by KIM
```

Comment:
you might want to check for `kf5-config` as well here, just in case this is no QT5 base installation.

Reply:
That's true.  But if no qt5/kf5 installed, it would be meaningless to install those desktop files in kservices5...

Reply:
That's right, it would just be some kind of defensive programming. I don't know whether this makes sense here.